docs/kubernetes-deployment/signer/README.md

```diff
@@ -29,23 +29,31 @@ ODIS signer is deployed in the following clusters:
 
 - Staging: cluster `integration-tests`
   - Signer0 in namespace `odis-signer0-staging` with this [`values-signer0-staging.yaml` file](./values/staging/values-signer0-staging.yaml).
-    - URL: `http://odis-signer0-staging.odis-signer0-staging:3000`. :warning: This is a URL internal to the cluster (i.e. not accessible from outside)!!
+    - URL internal: `http://odis-signer0-staging.odis-signer0-staging:3000`. :warning: This is a URL internal to the cluster (i.e. not accessible from outside)!!
```

Comment:
Is there any reason to keep both URL now, given that the external URL is the one that users and developperes will reference?

Reply:
I would say yes. From the point of view of the combiner:
- The internal URL is a direct connection (Combiner -> Signer).
- The external URL means going out of the cluster -> Outer internet -> cluster ingress (that is an NGIX) -> Signer.
So using the internal URL for the combiner is better/faster/less network hops.

It's true that users and developers cannot use the internal URL, but our combiner can take advantage of this internal URL :)

Reply:
Thanks for the input!